webapp/src/main/java/com/box/l10n/mojito/service/tm/search/TextUnitSearcher.java

```diff
@@ -256,7 +260,13 @@ NativeCriteria getCriteriaForSearch(TextUnitSearcherParameters searchParameters)
             .addProjection("a.path", "assetPath")
             .addProjection("atu.id", "assetTextUnitId")
             .addProjection("tu.created_date", "tmTextUnitCreatedDate")
-            .addProjection("atu.do_not_translate", "doNotTranslate"));
+            .addProjection("atu.do_not_translate", "doNotTranslate");
+
+    if (searchParameters.getStatusFilter() != null
+        && searchParameters.getStatusFilter().equals(StatusFilter.MT_TRANSLATED)) {
+      projection.addProjection("tptu.uploaded_file_uri", "uploadedFileUri");
```

Comment:
Leaving a note here to not forget about this projection - based on the outcome of the comment above.

Reply:
(There is also an update in `TextUnitDTONativeMapper` related to this too)